src/components/banners/SCSS/homeBanner.scss

```diff
@@ -21,6 +21,7 @@
       height: 100%;
       margin: 1rem auto;
       object-fit: contain;
+      min-height: 170px;
```

Comment:
maybe just change the size of the h1 of the banner
![Screenshot 2022-08-23 at 11 51 35](https://user-images.githubusercontent.com/22235112/186128951-b2979e9c-437d-4ce4-b1ef-74be69f75bbe.png)
![Screenshot 2022-08-23 at 11 51 53](https://user-images.githubusercontent.com/22235112/186128957-2cbeac7c-661e-4cc6-a8a5-2fced16e1e38.png)